src/PokemonGymImpl.java

```diff
@@ -0,0 +1,46 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class PokemonGymImpl implements PokemonGym {
+
+    private final String village;
+
+    public PokemonGymImpl(String village) {
+        this.village = village;
+        System.out.printf("The setting is a Pokemon Gym in the village of %s.\n", village);
+    }
+
```

Comment:
Hier hadden de implementaties gemoeten van de methodes die gedefinieerd staan in de PokemonGym interface